README.md

```diff
@@ -73,6 +91,15 @@ system.
 | [C++](onnxruntime/core/session/inference_session.h) | [Build from source](https://github.com/Microsoft/onnxruntime/blob/master/BUILD.md) | [Build from source](https://github.com/Microsoft/onnxruntime/blob/master/BUILD.md) |
 
 For builds using other execution providers, see Build Details below.
+=======
+* The GPU builds require the CUDA9.1 and cuDNN 7.3 runtime libraries being installed in the system.
+* Certain operators makes use of system locales. At the very least you will need to install English language package and configure en_US.UTF-8 locale.
+  * For Ubuntu install language-pack-en package
```

Comment:
do you know if this can be taken care of using the python requirements.txt file?